### ISSUE TITLE
When i use Web Worker, i get 'window is not defined'.

### ISSUE THREAD

**Issue by Quska45** [chartjs/chartjs-plugin-zoom#738]
Hi. i really enjoy your work in chartjs zoom plugin. Thank you.

But i have some ploblem use web worker.

When i use Web Worker, i get 'window is not defined'.

You have some idea this situation?

I use vite.

**Comment by LeeLenaleee** [comment]
This is correct, the window object is not available in web workers, which version of chart.js and zoom plugin are you using?

**Comment by Quska45** [comment]
Thank you for reply.
I think that you mean zoom plugin not available In web worker and offscreen both.
If I tried to use only offscreen, but even then, zoom plugin will not work?
I want to handle huge data, so i really need web worker and offscreen.
If you dont mind, please recommend some advice for this.

**Comment by LeeLenaleee** [comment]
If you want to use  offscreen canvas API (which you need in webworkers) you can't use the zoom plugin, so you will have to choose which one you want.

You can take a look at this page for some performance tips: https://www.chartjs.org/docs/master/general/performance.html
